apis/python/src/tiledb/vector_search/ingestion.py

```diff
@@ -593,7 +593,6 @@ def create_partial_write_array_group(
             ids_array_dom = tiledb.Domain(ids_array_rows_dim)
             ids_attr = tiledb.Attr(
                 name="values",
-                # TODO(paris): Enable configuring this so Vamana will work with different ID types.
```

Comment:
We'll need to change this when we support different ID types, but it's not just specific to Vamana, so removing.